### PR TITLE
boards: mec172xevb_assy6906: support remote flashing

### PR DESCRIPTION
**Pull request by dcpleung**
This allows mec172xevb_assy6906 to support misc-flasher to run arbitrary script to flash the SPI chip. A script is also provided as an example.

Signed-off-by: Daniel Leung <daniel.leung@intel.com>

### CODE REVIEW

File: boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (stephanosio) [inline boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh:1]: This is not going to work on Windows.

**Review by stephanosio — CHANGES_REQUESTED** [review]

File: boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh (hunk 1 above)
Comment (dcpleung) [reply boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh]: Updated to `#!/usr/bin/env bash`. Is this what you are asking for?

**Comment by dcpleung** [comment]
@stephanosio ping...

File: boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh (hunk 1 above)
Comment (stephanosio) [reply boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh]: > Updated to `#!/usr/bin/env bash`. Is this what you are asking for?

No, I am saying this shell script is not going to be able to run on Windows and the script should be written in, for example, Python.

Comment (dcpleung) [reply boards/arm/mec172xevb_assy6906/support/mec172x-remote-flasher.sh]: Understood. Let me see what I can do.

**Comment by dcpleung** [comment]
Updated this to be a Python script.

**Comment by dcpleung** [comment]
@stephanosio ping?

**Review by stephanosio — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**